### DISCUSSION TITLE
Java Agent not in latest release 8.8.1 tar file

### DISCUSSION THREAD

**Discussion by unc1** [apache/skywalking#7862] [General]
I downloaded the latest version 8.8.1 and didn't see the agent folder in the extracted contents. Was it removed? I have 8.7.0 and the agent directory was part of the extracted contents. Thanks.

**Comment by unc1** [comment]
Disregard this post. I just saw that there wasn't an 8.8.1 release of the Java agent.

**Reply by wu-sheng** [reply]
Yes, java agent release is separate now.

**Reply by MrZhengliang** [reply]
> Yes, java agent release is separate now.

Are Java agents independent in version 8.7.0 +, where can I get the agent directory?

**Reply by wu-sheng** [reply]
Read pinned discussion. All context is there.

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking/discussions/7548
